### ISSUE TITLE
Unable to retrieve password from the keychain store.

### ISSUE THREAD

**Issue by SK1724** [joeferraro/MavensMate-SublimeText#729]
Hi Joe,
I am running sublime on Win 7, 64 bit.
I tried the steps as you have mentioned in the forum (https://github.com/joeferraro/MavensMate-SublimeText/issues/228) regarding the issue of unable to retrieve pwd from Keychain but I am still getting the below error.

Operation: Your MavensMate project needs to be upgraded. Opening the upgrade UI.
Timestamp: Mon, 19 Oct 2015 13:12:16
   Result: [OPERATION FAILED]: Unable to retrieve password from the keychain store.

I also tried the following:
1. uninstalling Mavensmate
2. creating the new project after deletion of config directory
3. upgraded Mavensmate

Could you please help as I am unable to proceed.

Regards,
SK

**Comment by SK1724** [comment]
Hi Joe,
Closing this issue as found a way which resolved this!

I was performing the below steps earlier:
a. Deleted the config file on the existing project.
b. Then right clicked on the existing project and selected Mavensmate > Create New Project
c. After the Mavensmate UI opened, entered the credentials for the same project.
d. After Mavensmate has created the existing project (I should say -- rather updated the existing project :), I tried saving but it was giving the same error.

I closed the Mavensmate app and Sublime.
Then again opened Mavensmate and the existing project.
Wow.. It worked!!

Though I am facing an another issue as the save operation is now pretty slow.. taking 8-9 mins for a simple save.
But I am closing this issue.

Regards,
SK
